app/views/support_interface/support_users/index.html.erb

```diff
@@ -37,6 +38,14 @@
         <td class='govuk-table__cell'>
           <%= support_user.dfe_sign_in_uid %>
         </td>
+        <td class='govuk-table__cell'>
+          <%= link_to(
```

Comment:
```suggestion
          <%= govuk_link_to(
```

Reply:
@fofr sadly `govuk_link_to` doesn't accept a block, we'd need to update this helper to have the same functionality / signature as the ActionView `link_to` helper.